test/config/FeatureSwitchesSpec.scala

```diff
@@ -67,38 +67,4 @@ class FeatureSwitchesSpec extends UnitSpec {
 
   }
 
-  "FeatureSwitches" should {
-    "return true" when {
-      "the feature switch is set to true" in {
-        val config = Configuration(
-          "cl402.enabled" -> true
-        )
-
-        val featureSwitches = FeatureSwitches(config)
-
-        featureSwitches.isEnabled("cl402") shouldBe true
-      }
-
-      "the feature switch is not present in the config" in {
-        val config = Configuration.empty
-
-        val featureSwitches = FeatureSwitches(config)
-
-        featureSwitches.isEnabled("cl402") shouldBe true
-      }
-    }
-
-    "return false" when {
-      "the feature switch is set to false" in {
-        val config = Configuration(
-          "cl402.enabled" -> false
-        )
-
-        val featureSwitches = FeatureSwitches(config)
-
-        featureSwitches.isEnabled("cl402") shouldBe false
-      }
-    }
-  }
-
```

Comment:
This seems to be testing the Featureswitch config functionality and not specifically the `cl402` stuff. I would say keep this test in, but just make up a feature switch to load into config  on line 74. Something like 
`"test-feature-switch.enabled" -> true`